### DISCUSSION TITLE
Grab the current span

### DISCUSSION THREAD

**Discussion by 478467337** [open-telemetry/opentelemetry-java-instrumentation#5495] [Q&A]
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/manual-instrumentation.md#adding-attributes-to-the-current-span
Span.current().getSpanContext().getSpanId() = 0000000000000000

**Comment by mateuszrzeszutek** [comment]
Hey @478467337,

I suppose you're using the agent and you can't get the current span, is that correct? Can you post some more details? E.g. agent and opentelemetry-annotations versions that you're using, a code sample and/or a repo.

**Reply by 478467337** [reply]
delete  export OTEL_JAVAAGENT_EXCLUDE_CLASSES="io.opentelemetry.exporter.otlp.internal.*,io.opentelemetry.*"

**Reply by mateuszrzeszutek** [reply]
Sorry, I fail to understand your comment. Can you rephrase it?
